### ISSUE TITLE
Missing export functionality for the results of fit_scan_sequence() 

### ISSUE THREAD

**Issue by AvonReppert** [dschick/pyEvalData#37]
The built-in data analysis via the fit_scan_sequence() is very nice. 

For the comparison of the results of different scan_sequences() it would be useful to have the option to store previous results either as a human readable .txt file or in a nexus file. From this repository one could then load the results for further analysis and comparison to other datasets.

**Comment by dschick** [comment]
are you thinking of the actual evaluated data from `plot_scan_sequence()` or are we talking about the fit results from `fit_scan_sequence()`?

For option one, I do not really see so much gain from it, as the evaluation (reading, binning, averaging) is usually fast. Of course this might be different e.g. for 2D data.

But for option 2, I got the use case as some fitting routines are rather time consuming and you do not want to run them every time you start your evaluation.

I could think of some kind of caching algorithm, which generates a `hash` from the input data and fit model+parameters.
If enabled, the results will be saved to disk (file path is configurable; file_name will be the `hash` + prefix).
On the next evaluation, the `fit_scan_sequence()` method will first check for the results on disk before running the actual fits.
Similar to the [ukdm1Dsim](https://github.com/dschick/udkm1Dsim) there would be a switch, to re-evaluate the fits in any case.

**Comment by AvonReppert** [comment]
Yes Option 2, i.e. storing the fit results of a `fit_scan_sequence()` is what I have in mind. 
This would relax the need to rerun the fitting procedure everytime a kernel is re-started.

**Comment by dschick** [comment]
Are you fine with a hash and..npy file based solution as described above?

**Comment by AvonReppert** [comment]
Yes that would work.
